### PR TITLE
Started routes/recipes

### LINKED ISSUE #17: Get recipe by id
Work in source/backend/routes/recipes.js

Write a function that gets the recipe by id

GET /recipes/:recipeId
Search in database for the recipe with the given recipeId (should only return 1 object).

### PR DESCRIPTION
**Pull request by demyinn00**
# Resolves #17 
After creating your PR, please complete this template!
Cole did most of this function, I am just creating the PR. 
### What were your changes
- Added a recipe model {getRecipeById}
- Added a route to get the recipe by id inside of source/backend/routes/recipes.js

### What specifically should be checked
1. Check that the recipe model is correct
2. Check that the get recipe by id route is correct

### CODE REVIEW

File: source/backend/routes/recipes.js [hunk 1]
```diff
@@ -1,10 +1,32 @@
 const express = require('express');
+const recipesModel = require('../database/models/recipesModel');
 
 const router = express.Router();
 
 /* GET /recipes */
-router.get('/', (req, res) => {
-  res.send('recipes');
+router.get('/:id', async (req, res) => {
+  res.send('chicken');
```
Comment (mingkaili) [inline source/backend/routes/recipes.js:8]: Maybe remove this?

File: source/backend/routes/recipes.js [hunk 2]
```diff
@@ -1,10 +1,32 @@
...
+  try {
+    const { recipeId } = req.params.getByRecipeId;
+    //if (parseInt(req.params.id, 10) !== recipeId) {
```
Comment (mingkaili) [inline source/backend/routes/recipes.js:11]: We can probably remove the comments here too

**Review by mingkaili — APPROVED** [review]
Looks great! just 2 tiny things

File: source/backend/routes/recipes.js (hunk 1 above)
Comment (demyinn00) [reply source/backend/routes/recipes.js:8]: HAHAH yes! will make that adjustment

File: source/backend/routes/recipes.js (hunk 2 above)
Comment (demyinn00) [reply source/backend/routes/recipes.js:11]: Awesome will do

**Pull request merged**